src/dlgPackageExporter.cpp

```diff
@@ -1044,8 +1063,11 @@ void dlgPackageExporter::listTriggers()
         QStringList sl;
         sl << pChild->getName();
         auto pItem = new QTreeWidgetItem(sl);
-        pItem->setFlags(Qt::ItemIsUserCheckable | Qt::ItemIsAutoTristate | Qt::ItemIsEnabled | Qt::ItemIsSelectable);
+        pItem->setFlags(Qt::ItemIsUserCheckable | Qt::ItemIsEnabled | Qt::ItemIsSelectable);
         pItem->setCheckState(0, Qt::Unchecked);
+        if (pChild->isFolder() && pChild->getScript().isEmpty()) {
```

Comment:
how would this treat multiple nested empty folders? What if that was a purposeful decision?